Python module version constraints pose unnecessary platform limitations

Pandas 2.2.2 fails to install when using python >= 13
```
Collecting pandas==2.2.2 (from -r requirements.txt (line 11))
  Using cached pandas-2.2.2.tar.gz (4.4 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  � Preparing metadata (pyproject.toml) did not run successfully.
  ? exit code: 1
  ??> [170 lines of output]
      + meson setup /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/.mesonpy-_f7x25c6/build -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md --vsenv --native-file=/tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/.mesonpy-_f7x25c6/build/meson-python-native-file.ini
      The Meson build system
      Version: 1.2.1
      Source dir: /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a
      Build dir: /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/.mesonpy-_f7x25c6/build
      Build type: native build
      Project name: pandas
      Project version: 2.2.2
      C compiler for the host machine: cc (gcc 13.3.0 "cc (Ubuntu 13.3.0-6ubuntu2~24.04.1) 13.3.0")
      C linker for the host machine: cc ld.bfd 2.42
      C++ compiler for the host machine: c++ (gcc 13.3.0 "c++ (Ubuntu 13.3.0-6ubuntu2~24.04.1) 13.3.0")
      C++ linker for the host machine: c++ ld.bfd 2.42
      Cython compiler for the host machine: cython (cython 3.0.5)
      Host machine cpu family: x86_64
      Host machine cpu: x86_64
      Program python found: YES (/home/example_user/miniconda3/envs/holdem14/bin/python3.14)
      Found pkg-config: /usr/bin/pkg-config (1.8.1)
      Run-time dependency python found: YES 3.14
      Build targets in project: 53
      
      pandas 2.2.2
      
        User defined options
          Native files: /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/.mesonpy-_f7x25c6/build/meson-python-native-file.ini
          buildtype   : release
          vsenv       : True
          b_ndebug    : if-release
          b_vscrt     : md
      
      Found ninja-1.13.0.git.kitware.jobserver-pipe-1 at /tmp/pip-build-env-kzf8e937/normal/bin/ninja
      
      Visual Studio environment is needed to run Ninja. It is recommended to use Meson wrapper:
      /tmp/pip-build-env-kzf8e937/overlay/bin/meson compile -C .
      + /tmp/pip-build-env-kzf8e937/normal/bin/ninja
      [1/151] Generating pandas/_libs/khash_primitive_helper_pxi with a custom command
      [2/151] Generating pandas/_libs/hashtable_func_helper_pxi with a custom command
      [3/151] Generating pandas/_libs/intervaltree_helper_pxi with a custom command
      [4/151] Generating pandas/_libs/algos_take_helper_pxi with a custom command
      [5/151] Generating pandas/_libs/hashtable_class_helper_pxi with a custom command
      [6/151] Generating pandas/_libs/sparse_op_helper_pxi with a custom command
      [7/151] Generating pandas/_libs/algos_common_helper_pxi with a custom command
      [8/151] Generating pandas/_libs/index_class_helper_pxi with a custom command
      [9/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/ccalendar.pyx
      [10/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/base.pyx
      [11/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/np_datetime.pyx
      [12/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/nattype.pyx
      warning: /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/nattype.pyx:79:0: Global name __nat_unpickle matched from within class scope in contradiction to to Python 'class private name' rules. This may change in a future release.
      warning: /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/nattype.pyx:79:0: Global name __nat_unpickle matched from within class scope in contradiction to to Python 'class private name' rules. This may change in a future release.
      [13/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/dtypes.pyx
      [14/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/arrays.pyx
      [15/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/vectorized.pyx
      [16/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/timezones.pyx
      [17/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/fields.pyx
      [18/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/indexing.pyx
      [19/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/conversion.pyx
      [20/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/strptime.pyx
      [21/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/tzconversion.pyx
      [22/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/timestamps.pyx
      [23/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/parsing.pyx
      [24/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/offsets.pyx
      [25/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/properties.pyx
      [26/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/hashing.pyx
      [27/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/timedeltas.pyx
      [28/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/ops_dispatch.pyx
      [29/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslibs/period.pyx
      [30/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/internals.pyx
      [31/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/missing.pyx
      [32/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/byteswap.pyx
      [33/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/ops.pyx
      [34/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/testing.pyx
      [35/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/reshape.pyx
      [36/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/index.pyx
      [37/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/sas.pyx
      [38/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/parsers.pyx
      [39/151] Compiling C object pandas/_libs/tslibs/base.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_base.pyx.c.o
      [40/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/tslib.pyx
      [41/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/lib.pyx
      [42/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/writers.pyx
      [43/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/window/indexers.pyx
      [44/151] Compiling C object pandas/_libs/tslibs/parsing.cpython-314-x86_64-linux-gnu.so.p/.._src_parser_tokenizer.c.o
      [45/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/interval.pyx
      [46/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/join.pyx
      [47/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/window/aggregations.pyx
      [48/151] Compiling C object pandas/_libs/tslibs/ccalendar.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_ccalendar.pyx.c.o
      [49/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/sparse.pyx
      [50/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/hashtable.pyx
      [51/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/algos.pyx
      [52/151] Compiling C object pandas/_libs/tslibs/np_datetime.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_np_datetime.pyx.c.o
      [53/151] Compiling Cython source /tmp/pip-install-zlqbl57v/pandas_03444f84fc444088959fa4eb9f96578a/pandas/_libs/groupby.pyx
      [54/151] Compiling C object pandas/_libs/tslibs/nattype.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_nattype.pyx.c.o
      [55/151] Compiling C object pandas/_libs/tslibs/dtypes.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_dtypes.pyx.c.o
      [56/151] Compiling C object pandas/_libs/arrays.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_arrays.pyx.c.o
      [57/151] Compiling C object pandas/_libs/tslibs/conversion.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_conversion.pyx.c.o
      [58/151] Compiling C object pandas/_libs/indexing.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_indexing.pyx.c.o
      [59/151] Compiling C object pandas/_libs/tslibs/vectorized.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_vectorized.pyx.c.o
      [60/151] Compiling C object pandas/_libs/hashing.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_hashing.pyx.c.o
      [61/151] Compiling C object pandas/_libs/tslibs/timezones.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_timezones.pyx.c.o
      [62/151] Compiling C object pandas/_libs/tslibs/tzconversion.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_tzconversion.pyx.c.o
      [63/151] Compiling C object pandas/_libs/pandas_datetime.cpython-314-x86_64-linux-gnu.so.p/src_vendored_numpy_datetime_np_datetime.c.o
      [64/151] Compiling C object pandas/_libs/tslibs/fields.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_fields.pyx.c.o
      [65/151] Compiling C object pandas/_libs/lib.cpython-314-x86_64-linux-gnu.so.p/src_parser_tokenizer.c.o
      [66/151] Compiling C object pandas/_libs/pandas_datetime.cpython-314-x86_64-linux-gnu.so.p/src_datetime_date_conversions.c.o
      [67/151] Compiling C object pandas/_libs/pandas_datetime.cpython-314-x86_64-linux-gnu.so.p/src_vendored_numpy_datetime_np_datetime_strings.c.o
      [68/151] Compiling C object pandas/_libs/pandas_datetime.cpython-314-x86_64-linux-gnu.so.p/src_datetime_pd_datetime.c.o
      [69/151] Compiling C object pandas/_libs/pandas_parser.cpython-314-x86_64-linux-gnu.so.p/src_parser_io.c.o
      [70/151] Compiling C object pandas/_libs/pandas_parser.cpython-314-x86_64-linux-gnu.so.p/src_parser_pd_parser.c.o
      [71/151] Compiling C object pandas/_libs/pandas_parser.cpython-314-x86_64-linux-gnu.so.p/src_parser_tokenizer.c.o
      [72/151] Compiling C object pandas/_libs/parsers.cpython-314-x86_64-linux-gnu.so.p/src_parser_io.c.o
      [73/151] Compiling C object pandas/_libs/json.cpython-314-x86_64-linux-gnu.so.p/src_vendored_ujson_python_ujson.c.o
      [74/151] Compiling C object pandas/_libs/parsers.cpython-314-x86_64-linux-gnu.so.p/src_parser_tokenizer.c.o
      [75/151] Compiling C object pandas/_libs/json.cpython-314-x86_64-linux-gnu.so.p/src_vendored_ujson_python_JSONtoObj.c.o
      [76/151] Compiling C object pandas/_libs/json.cpython-314-x86_64-linux-gnu.so.p/src_vendored_ujson_lib_ultrajsonenc.c.o
      [77/151] Compiling C object pandas/_libs/json.cpython-314-x86_64-linux-gnu.so.p/src_vendored_ujson_python_objToJSON.c.o
      [78/151] Compiling C object pandas/_libs/tslibs/strptime.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_strptime.pyx.c.o
      [79/151] Compiling C object pandas/_libs/json.cpython-314-x86_64-linux-gnu.so.p/src_vendored_ujson_lib_ultrajsondec.c.o
      [80/151] Compiling C object pandas/_libs/tslibs/parsing.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_parsing.pyx.c.o
      [81/151] Compiling C object pandas/_libs/ops_dispatch.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_ops_dispatch.pyx.c.o
      [82/151] Compiling C object pandas/_libs/missing.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_missing.pyx.c.o
      [83/151] Compiling C object pandas/_libs/properties.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_properties.pyx.c.o
      [84/151] Compiling C object pandas/_libs/tslibs/period.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_period.pyx.c.o
      [85/151] Compiling C object pandas/_libs/byteswap.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_byteswap.pyx.c.o
      [86/151] Compiling C object pandas/_libs/internals.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_internals.pyx.c.o
      [87/151] Compiling C object pandas/_libs/tslibs/timestamps.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_timestamps.pyx.c.o
      [88/151] Compiling C object pandas/_libs/testing.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_testing.pyx.c.o
      [89/151] Compiling C object pandas/_libs/ops.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_ops.pyx.c.o
      [90/151] Compiling C++ object pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_window_aggregations.pyx.cpp.o
      FAILED: [code=1] pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_window_aggregations.pyx.cpp.o
      c++ -Ipandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p -Ipandas/_libs/window -I../../pandas/_libs/window -I../../../../pip-build-env-kzf8e937/overlay/lib/python3.14/site-packages/numpy/_core/include -I../../pandas/_libs/include -I/home/example_user/miniconda3/envs/holdem14/include/python3.14 -fvisibility=hidden -fvisibility-inlines-hidden -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -w -O3 -DNPY_NO_DEPRECATED_API=0 -DNPY_TARGET_VERSION=NPY_1_21_API_VERSION -fPIC -MD -MQ pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_window_aggregations.pyx.cpp.o -MF pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_window_aggregations.pyx.cpp.o.d -o pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_window_aggregations.pyx.cpp.o -c pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:422:31: error: standard attributes in middle of decl-specifiers
        422 |         #define CYTHON_UNUSED [[maybe_unused]]
            |                               ^
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:2260:12: note: in expansion of macro ?CYTHON_UNUSED?
       2260 |     static CYTHON_UNUSED PyObject *__Pyx_KwargsAsDict_FASTCALL(PyObject *kwnames, PyObject *const *kwvalues);
            |            ^~~~~~~~~~~~~
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:422:31: note: standard attributes must precede the decl-specifiers to apply to the declaration, or follow them to apply to the type
        422 |         #define CYTHON_UNUSED [[maybe_unused]]
            |                               ^
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:2260:12: note: in expansion of macro ?CYTHON_UNUSED?
       2260 |     static CYTHON_UNUSED PyObject *__Pyx_KwargsAsDict_FASTCALL(PyObject *kwnames, PyObject *const *kwvalues);
            |            ^~~~~~~~~~~~~
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:422:31: error: standard attributes in middle of decl-specifiers
        422 |         #define CYTHON_UNUSED [[maybe_unused]]
            |                               ^
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:42115:8: note: in expansion of macro ?CYTHON_UNUSED?
      42115 | static CYTHON_UNUSED PyObject *__Pyx_KwargsAsDict_FASTCALL(PyObject *kwnames, PyObject *const *kwvalues) {
            |        ^~~~~~~~~~~~~
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:422:31: note: standard attributes must precede the decl-specifiers to apply to the declaration, or follow them to apply to the type
        422 |         #define CYTHON_UNUSED [[maybe_unused]]
            |                               ^
      pandas/_libs/window/aggregations.cpython-314-x86_64-linux-gnu.so.p/pandas/_libs/window/aggregations.pyx.cpp:42115:8: note: in expansion of macro ?CYTHON_UNUSED?
      42115 | static CYTHON_UNUSED PyObject *__Pyx_KwargsAsDict_FASTCALL(PyObject *kwnames, PyObject *const *kwvalues) {
            |        ^~~~~~~~~~~~~
      [91/151] Generating pandas/__init__.py with a custom command
      [92/151] Compiling C object pandas/_libs/tslibs/timedeltas.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_timedeltas.pyx.c.o
      [93/151] Compiling C object pandas/_libs/reshape.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_reshape.pyx.c.o
      [94/151] Compiling C object pandas/_libs/sas.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_sas.pyx.c.o
      [95/151] Compiling C object pandas/_libs/tslib.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslib.pyx.c.o
      [96/151] Compiling C object pandas/_libs/writers.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_writers.pyx.c.o
      [97/151] Compiling C object pandas/_libs/window/indexers.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_window_indexers.pyx.c.o
      [98/151] Compiling C object pandas/_libs/parsers.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_parsers.pyx.c.o
      [99/151] Compiling C object pandas/_libs/lib.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_lib.pyx.c.o
      [100/151] Compiling C object pandas/_libs/index.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_index.pyx.c.o
      [101/151] Compiling C object pandas/_libs/tslibs/offsets.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_tslibs_offsets.pyx.c.o
      [102/151] Compiling C object pandas/_libs/sparse.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_sparse.pyx.c.o
      [103/151] Compiling C object pandas/_libs/interval.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_interval.pyx.c.o
      [104/151] Compiling C object pandas/_libs/join.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_join.pyx.c.o
      [105/151] Compiling C object pandas/_libs/hashtable.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_hashtable.pyx.c.o
      [106/151] Compiling C object pandas/_libs/algos.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_algos.pyx.c.o
      [107/151] Compiling C object pandas/_libs/groupby.cpython-314-x86_64-linux-gnu.so.p/meson-generated_pandas__libs_groupby.pyx.c.o
      ninja: build stopped: subcommand failed.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

� Encountered error while generating package metadata.
??> pandas

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```
Pytorch ROCm support  does not exist in version 2.5.1 which excludes amd gpu usage, severely impacting potential performance in some systems.

See also https://github.com/Reinforcement-Poker/pokers/issues/5 regarding pokers

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-torch==2.5.1
+torch>=2.5.1,<3.0
 numpy==1.26.4
 matplotlib==3.9.0
 tensorboard==2.18.0
@@ -8,5 +8,5 @@ pokers @ git+https://github.com/dberweger2017/pokers.git@b1a48bd2067176cb73f9528
 python-dotenv==1.0.1
 requests==2.32.3
 seaborn==0.13.2
-pandas==2.2.2
+pandas>=2.2.2,<3.0
 PyQt5==5.15.11

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,8 @@
 torch>=2.5.1,<3.0
-numpy==1.26.4
-matplotlib==3.9.0
+numpy==1.26.4; python_version < "3.13"
+numpy>=2.1,<3.0; python_version >= "3.13"
+matplotlib==3.9.0; python_version < "3.13"
+matplotlib>=3.10.7,<3.11; python_version >= "3.13"
 tensorboard==2.18.0
 argparse
 tqdm==4.67.0
@@ -8,5 +10,5 @@ pokers @ git+https://github.com/dberweger2017/pokers.git@b1a48bd2067176cb73f9528
 python-dotenv==1.0.1
 requests==2.32.3
 seaborn==0.13.2
-pandas>=2.2.2,<3.0
+pandas>=2.3.3,<3.0
 PyQt5==5.15.11

